app/scripts/modules/core/src/pipeline/config/stages/monitoreddeploy/DeploymentMonitorExecutionDetails.tsx

```diff
@@ -70,6 +73,11 @@ function getDeploymentMonitorAdditionalDetails(stage: IExecutionStage) {
   return null;
 }
 
+function isRunningOrComplete(stage: IExecutionStage) {
+  const runOrCompleteStatuses = ['RUNNING', 'SUCCEEDED', 'TERMINAL', 'FAILED_CONTINUE', 'PAUSED', 'CANCELED'];
```

Comment:
What statuses are you trying to guard against?  is it just `NOT_STARTED`, or does it include others?  If we inverted this status check, what would it test?

I am not very comfortable with assigning semantic meaning to lists (i.e., this list is "Running OR Complete") outside of a centralized area.  It worries me because we start to have different answers for "Is this stage complete?" in different areas of the app.  

Unfortunately, deck doesn't currently have centralized logic for testing these statuses, so there's no blaming here.  I'm just voicing my concern.

Reply:
TOTALLY agree, and I was sort of sad writing this... I guess I can use `hasNotStarted` from the transformer @erikmunson mentioned, how do you feel about that?

Reply:
yeah, I think you can simply call `stage.hasNotStarted()`, neat